client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningCheck.tsx

```diff
@@ -172,32 +165,34 @@ const ContactQuestioningCheck: React.FC<Props> = (props: Props): JSX.Element =>
                     />
                 </Grid>
                 <Grid item>
-                    <Grid container justify='space-between'>
-                        <FieldName xs={5} fieldName={OCCUPATION_LABEL}/>
-                        <Grid item xs={7}>
+                    <Grid container>
+                        <FieldName xs={5} fieldName={ContactQuestioningFieldsNames.OCCUPATION} className={classes.fieldName}/>
+                        <Grid item xs={5}>
                         <Controller
                             control={control}
                             name={`form[${index}.${InteractedContactFields.OCCUPATION}]`}
                             defaultValue={interactedContact.occupation}
                             render={(props) => {
-                                return (<FormControl variant='outlined' fullWidth>
-                                    <Select
-                                        {...props}
-                                        placeholder={OCCUPATION_LABEL}
-                                        onChange={(event) => {
-                                            props.onChange(event.target.value)
-                                        }}>
-                                        {
-                                           occupations?.length > 0 && occupations.map((occupation) => (
-                                                <MenuItem className={classes.menuItem}
-                                                    key={occupation}
-                                                    value={occupation}>
-                                                    {occupation}
-                                                </MenuItem>
-                                            ))
-                                        }
-                                    </Select>
-                                </FormControl>)
+                                return (
+                                    <FormControl variant='outlined' fullWidth>
+                                        <Select
+                                            {...props}
+                                            placeholder={ContactQuestioningFieldsNames.OCCUPATION}
+                                            onChange={(event) => {
+                                                props.onChange(event.target.value)
```

Comment:
Can you use event.target?.value ? If possible(With the question mark) , Im not certain but I remember once it crashed the client on a bug I worked on